src/app/public/modules/inline-delete/fixtures/inline-delete=fixtures.module.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  CommonModule
+} from '@angular/common';
+
+import {
+  InlineDeleteTestComponent
+} from './inline-delete.component.fixture';
+
+import {
+  SkyInlineDeleteModule
+} from '../';
+
+@NgModule({
+  declarations: [
+    InlineDeleteTestComponent
+  ],
+  imports: [
+    CommonModule,
+    SkyInlineDeleteModule
+  ],
+  exports: [
+    InlineDeleteTestComponent
+  ]
+})
+export class SkyInlineDeleteFixturesModule { }
```

Comment:
Looks like your file name got a `=` in it?

Reply:
Done